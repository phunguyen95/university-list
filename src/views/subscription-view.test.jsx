import { render, screen } from "@testing-library/react";
import SubscriptionView from "./subcription-view";
import { Router } from "react-router-dom";
import { createMemoryHistory } from "history";

test("renders error view without crashing", () => {
    const history = createMemoryHistory();

  const { asFragment } = render(<Router history={history}><SubscriptionView /></Router>);

  expect(asFragment()).toMatchSnapshot();
});
