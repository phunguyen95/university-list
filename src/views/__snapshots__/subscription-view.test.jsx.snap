// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders error view without crashing 1`] = `
<DocumentFragment>
  <div
    class="bd-newsletter"
    id="newsletter"
  >
    <div
      class="Toastify"
    />
    <section
      class="bd-index-section bd-newsletter-box"
    >
      <div
        class="bd-newsletter-heading"
      >
        <h2
          class="title has-text-black mb-0 is-size-2-widescreen"
        >
          <span
            class="icon has-text-primary is-size-2-widescreen smallIcon"
          >
             
            <i
              class="fas fa-paper-plane"
            />
             
          </span>
          <strong
            class="heading-newsletter"
          >
            Newsletter
          </strong>
          <br />
          <small
            class="subheading"
          >
            Features, releases, showcaseâ€¦ stay in the loop!
          </small>
           
        </h2>
        <div
          class="bd-newsletter-fields"
        >
          <div
            class="control has-icons-left is-expanded"
          >
            <input
              class="input is-medium is-primary"
              name="email"
              placeholder="email address"
              required=""
              type="email"
              value=""
            />
            <span
              class="icon is-small is-left"
            >
              <i
                class="fas fa-envelope"
              />
               
            </span>
            <img
              class="spam-free"
              height="88"
              src="https://bulma.io/images/drawing/spam-free.png"
              width="112"
            />
          </div>
          <div
            class="control subscribe-button"
          >
            <button
              class="button is-medium is-primary subscribe-button"
            >
              <strong>
                Subscribe
              </strong>
               
            </button>
          </div>
        </div>
      </div>
    </section>
  </div>
</DocumentFragment>
`;
